Fixes #23580: Be more precise in the 8.0.0 release notes (#785)
Normation/rudder-tools: release-data/changelogs/8.0/index.adoc

* Added: Debian 12
* Now require extended subscription: SLES 11 \<=SP4, SLES 12 \<=SP4, SLES 15 \<=SP2, Windows Server 2008R2, 2012 and 2012R2, Ubuntu 16.04 LTS, Debian 9

> * Rudder agents older than 7.3 will not be supported anymore

Reviewer: ```suggestion
* Rudder agents older than 7.3 are not supported on 8.0 servers
```

> * Rudder agents older 7.0 will not be able to communicate at all with a 8.0.X Rudder server

Reviewer: ```suggestion
```

The YAML format also becomes the internal storage format for techniques (used in the configuration
git and archives).

> This format is stable, so techniques can be shared between Rudder servers having higher than 8.0 versions, even if differents.

Reviewer: ```suggestion
This format is stable, so techniques can be shared between Rudder servers having higher than 8.0 versions, even if different.
```
